react/components/SearchResult.js

```diff
@@ -34,9 +37,12 @@ export default class SearchResult extends Component {
 
   handleLayoutChange = (e, mode) => {
     e.preventDefault()
+    const defaultModes = ['small', 'inline', 'normal']
+    const modeIndex = (defaultModes.indexOf(this.state.galleryLayoutMode) + 1) % 3
```

Comment:
Seems like a overkill since we are the ones defining the values you can pick for the mode, but ok, you can leave it this way

Reply:
It will be resolved when we pick the values dynamically via pages